Update 40_WYSIWYG.md (#16177)
pimcore/pimcore: doc/03_Documents/01_Editables/40_WYSIWYG.md

Similar to Textarea and Input you can use the WYSIWYG editable in the templates to provide rich-text editing. TinyMce is installed by default in our demo. Another editor can be installed via the wysiwyg-events you find in `events.js`

## Enable TinyMce

Add the bundle in your `config/bundles.php` file:

```php
\Pimcore\Bundle\TinymceBundle\PimcoreTinymceBundle::class => ['all' => true],
```

> After this be sure, that PimcoreTinymceBundle is installed. You can check it with: `bin/console pimcore:bundle:list`

Reviewer: ```suggestion
After this, make sure PimcoreTinymceBundle is installed with `bin/console pimcore:bundle:list`.
```

> If it is not installed, you can install it with: `bin/console pimcore:bundle:install PimcoreTinymceBundle`

Reviewer: ```suggestion
If it is not installed, you can install it with `bin/console pimcore:bundle:install PimcoreTinymceBundle`.
```
